testsuite/features/secondary/minkvm_guests.feature

```diff
@@ -342,6 +342,95 @@ Feature: Be able to manage KVM virtual machines via the GUI
     Then I wait until I do not see "test-net1" text
     And I should not see a "test-net1" virtual network on "kvm_server"
 
+# Start provisioning scenarios
+
+# This scenario needs the 15-SP2 base system channel to be synchronized
```

Comment:
If we need it, it might be interesting to have a step here waiting for that channel to be fully synched :thinking: 
Better than just guess than it had enough time to synch.

Reply:
We already wait for the channel sync in the step killing (or not) the repo syncs

Reply:
Which is in another feature file right? :) 
Better to add an extra pre-condition than to have dependencies with other feature files.

Reply:
Added